src/bitbucket/types.d.ts

```diff
@@ -44,6 +44,15 @@ declare namespace BB {
     url: string;
   };
 
+  type MergeStatusResponse =
+    | {
+        task_status: 'PENDING';
+      }
+    | {
+        task_status: 'SUCCESS';
+        merge_result: any;
```

Comment:
I guess this is fine for now ?

Reply:
Yeah was about to add an explanation. We don't currently make use of the merge_result for any type of merge so didn't even really need to include this field but thought I might as well so we know the general shape and can add typing to this if required later.

Reply:
if we have a ticket or task later for this or just a comment might be worth (not in this PR if you don't want)